ubuntu1-1/scripts/verify-env.sh

```
@@ -0,0 +1,240 @@
+
+# 2. Check Home Manager
+echo_header "Home Manager"
+check_command "home-manager" ".nix-profile"
+
+# 3. Check PATH priority (Nix should come before /usr/bin)
+echo_header "PATH Priority"
+if [[ "$PATH" == *".nix-profile/bin"* ]]; then
+    echo_success "Nix profile in PATH"
+    
+    # Check if Nix comes before /usr/bin
+    local nix_pos=$(echo $PATH | grep -bo ".nix-profile/bin" | cut -d: -f1)
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)